traversal/focus_test.go

```diff
@@ -123,6 +123,26 @@ func TestFocusSingleTree(t *testing.T) {
 	})
 }
 
+// covers Get used on one already-loaded Node; no link-loading exercised.
```

Comment:
a test that exercises an attempt to traverse through a link might be good, it would at least show what would be expected to occur in such a situation, I imagine it's some error that bubbles up from the bowels of `Link#Load()`?